compiler_gym/envs/llvm/service/Observation.cc

```diff
@@ -228,6 +229,38 @@ Status setObservation(LlvmObservationSpace space, const fs::path& workingDirecto
     case LlvmObservationSpace::BUILDTIME: {
       return benchmark.computeBuildtime(reply);
     }
+    case LlvmObservationSpace::LEXEDIR: {
```

Comment:
```suggestion
    case LlvmObservationSpace::LEXED_IR: {
```

Forgive my bikeshedding 🙂  LEXED_IR will translate to pascal case "LexedIr" on the python side